examples/DrawPrimitives/minimalradium.hpp

```diff
@@ -5,9 +5,6 @@
 #include <Engine/Scene/System.hpp>
 #include <Engine/Scene/SystemDisplay.hpp>
 
-/* This file contains a minimal radium/qt application which shows the
-classic "Spinning Cube" demo. */
-
 /// This is a very basic component which holds a spinning cube.
```

Comment:
Not sure this comment is correct. There is no spinning cube in the component but all the geometric primitives Radium can handle.